llm/llama-3/gui.yaml

```diff
@@ -0,0 +1,46 @@
+# Starts a GUI server that connects to the Llama-3 OpenAI API server.
+#
+# This works with the endpoint.yaml, please refer to llm/llama-3/README.md
+# for more details.
+#
+# Usage:
+#
+#  1. If you have a endpoint started on a cluster (sky launch):
+#     `sky launch -c llama3-gui ./gui.yaml --env ENDPOINT=$(sky status --ip llama3):8000`
```

Comment:
Should we change all usages of `sky status --ip` to `sky status --endpoint 8000 cluster` in this PR? `--endpoint` is more general and works for k8s as well.